update archive instructions since archiving with roles in the US is n… (#5378)
DataDog/documentation: content/en/logs/archives/s3.md

Go into your [AWS Console][2] and [create an S3 bucket][3] to send your archives to. Be careful not to make your bucket publicly readable. 

> Next, grant Datadog permissions to write log archives to your S3 bucket. Users of the US Site do this via role delegation, users of the EU Site do this via the bucket policy.

Reviewer: ```suggestion
Next, grant Datadog permissions to write log archives to your S3 bucket. US site users do this with role delegation. EU site users do this with the bucket policy.
```

{{< tabs >}}
{{% tab "US Site" %}}

1. Set up the [AWS integration][1] for the AWS account that holds your S3 bucket. This involves [creating a role][2] that Datadog can use to integrate with AWS Cloudwatch. 

> 2. Add the following permission statement to the IAM policies of your Datadog role (be sure to edit the bucket names and, if desired, specify the paths that contain your log archives). The `GetObject` and `ListBucket` permissions allow for [Rehydrating from Archives][3]. The `PutObject` permissions is sufficient for uploading archives.

Reviewer: ```suggestion
2. Add the following permission statement to the IAM policies of your Datadog role (be sure to edit the bucket names and, if desired, specify the paths that contain your log archives). The `GetObject` and `ListBucket` permissions allow for [Rehydrating from Archives][3]. The `PutObject` permission is sufficient for uploading archives.
```

}
```

> 

Reviewer: The code block has to be indented 4 spaces so it doesn't break the numbered list (see the preview).

3. Go to your [Pipelines page in Datadog][4], and select the **Add archive** option at the bottom. Only Datadog users with admin status can complete this and the following step.

4. Select the appropriate AWS account + role combination for your S3 bucket. Input your bucket name. Optionally input a prefix directory for all the content of your log archives. Save your archive, and you are finished. 

    {{< img src="logs/archives/log_archives_s3_datadog_settings_role_delegation.png" alt="Set your S3 bucket info in Datadog" responsive="true" style="width:75%;">}}
    
[1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
[2]: /integrations/amazon_web_services/?tab=allpermissions#installation
[3]: /logs/archives/rehydrating
[4]: https://app.datadoghq.com/logs/pipelines
{{% /tab %}}

{{% tab "EU Site" %}}

> 1. Modify your bucket to grant write-only access to the Datadog AWS user. Do this by editing your bucket's **permissions**, and setting the **bucket policy** with the following content (replace `<MY_BUCKET_NAME>` with the name of your bucket):

Reviewer: ```suggestion
1. Modify your bucket to grant write-only access to the Datadog AWS user by editing your bucket's **permissions**, and setting the **bucket policy** with the following content (update `<MY_BUCKET_NAME>`):
```

    ```
    {
        "Version": "2012-10-17",
        "Statement": [
            {
                "Sid": "AllowDatadogArchivesUploader",
                "Effect": "Allow",
                "Principal": {
                    "AWS": "464622532012"
                },
                "Action": [
                    "s3:PutObject",
                    "s3:PutObjectAcl",
                    "s3:ListMultipartUploadParts",
                    "s3:AbortMultipartUpload"
                ],
                "Resource": "arn:aws:s3:::<MY_BUCKET_NAME>/*"
            }
        ]
    }
    ```

>     {{< img src="logs/archives/log_archives_s3_iam_policy.png" alt="IAM Policy for S3 Archives" responsive="true" style="width:75%;">}}

Reviewer: There's a current bug with our site where images within tabs, within ordered lists will break the numbering. If this screenshoot is unnecessary, I suggest removing it.

Author: removing the screenshot or code bracket in this case is not an option. since this issue is already there, i'm ok with it staying.

2. Go to your [Pipelines page in Datadog][1], and select the **Add archive** option at the bottom. Only Datadog users with admin status can complete this and the following step.

> 3. Input your bucket name. Optionally input a prefix directory for all the content of your log archives. Save your archive, and you are finished. 

Reviewer: ```suggestion
3. Input your bucket name. Optionally input a prefix directory for all the content of your log archives. Save your archive.
```
